### ISSUE TITLE
Improve low-contrast text for accessibility

### ISSUE THREAD

**Issue by bdolor** [BCcampus/bcc-sage#72]
**What is the current behavior?**

Low-contrast text is difficult or impossible for many users to read. Color contrast on related links, tags, next and previous post links generate a failing grade on an accessibility audit

**What is the expected or desired behavior?**

Color contrast should be large enough (no blue on blue) for people to read.
ref: https://dequeuniversity.com/rules/axe/2.2/color-contrast

example: https://helga.bccampus.ca/wp/bcc/playing-with-the-possibilities-liberating-structures-online/
![image](https://user-images.githubusercontent.com/2048170/31837034-240fafc4-b58c-11e7-960c-a08399482b0e.png)
